### ISSUE TITLE
Tests do not run on Windows

### ISSUE THREAD

**Issue by k2snowman69** [snowcoders/sortier#642]
Tests aren't seemingly running on windows

```
D:\Projects\sortier\cli>npm run test

> @snowcoders/sortier@3.0.0 test D:\Projects\sortier\cli
> nyc npm run mocha


> @snowcoders/sortier@3.0.0 mocha D:\Projects\sortier\cli
> mocha --require ts-node/register --require source-map-support/register --full-trace "./src/**/*.test.ts"

internal/modules/cjs/loader.js:797
    throw err;
    ^

Error: Cannot find module 'D:\Projects\sortier\cli\node'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:794:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1025:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @snowcoders/sortier@3.0.0 mocha: `mocha --require ts-node/register --require source-map-support/register --full-trace "./src/**/*.test.ts"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @snowcoders/sortier@3.0.0 mocha script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\RickyPatel\AppData\Roaming\npm-cache\_logs\2019-11-11T00_16_01_060Z-debug.log

=============================== Coverage summary ===============================
Statements   : 0% ( 0/347 )
Branches     : 0% ( 0/172 )
Functions    : 0% ( 0/47 )
Lines        : 0% ( 0/343 )
================================================================================
```

**Comment by k2snowman69** [comment]
Issue seems to be nyc of all things

**Comment by k2snowman69** [comment]
https://github.com/snowcoders/sortier/pull/643

**Comment by k2snowman69** [comment]
Fixed and merged

**Comment by k2snowman69** [comment]
Also released 3.0.1 to fix those who are using these types
